packages/core/core/src/factories.ts

```diff
@@ -101,4 +115,4 @@ function createCoreRouter<T extends Common.UID.ContentType>(
   };
 }
 
-export { createCoreController, createCoreService, createCoreRouter };
+export { createCoreController, createCoreService, createCoreRouter, symbols };
```

Comment:
I feel like there has to be a better way than exporting the symbol here, because people will wonder what it is. Maybe move it to a constants file so it can be used in both places without being visible here? Or do you think that's overcomplicating it?

Reply:
Done in [18c84b0](https://github.com/strapi/strapi/pull/19061/commits/18c84b0537849bcbab4dca00c723a7c95317fa24)